qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -39,43 +44,43 @@ public void verifyModulesAvailableOnHomePage() {
         assertTrue(homePage.isSystemAdministrationAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on homepage after login as clerk")
     public void verifyClerkModulesAvailableOnHomePage() {
         goToLoginPage().loginAsClerk();
-        initiateHomePage();
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
         assertTrue(homePage.isRegisterPatientCustomizedForRefAppPresent());
+        assertFalse(homePage.isConfigureMetadataAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on home page after login as doctor")
     public void verifyDoctorModulesAvailableOnHomePage() {
         goToLoginPage().loginAsDoctor();
-        initiateHomePage();
         assertTrue(homePage.isFindAPatientAppPresent());
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
+        assertFalse(homePage.isConfigureMetadataAppPresent());
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on home page after login as nurse")
     public void verifyNurseModulesAvailableOnHomePage() {
         goToLoginPage().loginAsNurse();
-        initiateHomePage();
         assertTrue(homePage.isFindAPatientAppPresent());
         assertTrue(homePage.isActiveVisitsAppPresent());
```

Comment:
Is the nurse allowed to see the configure metadata app?

Reply:
Nurse is not allowed to see configure metadata app.